resources/js/controllers/fields/upload_controller.js

```diff
@@ -236,15 +248,41 @@ export default class extends Controller {
                     }
                 });
 
-                this.on('completemultiple', () => {
-                    //  $(`${dropname}.sortable-dropzone`).sortable('enable');
+                this.on("maxfilesexceeded", (file) => {
+                    console.log('dropzone.maxfilesexceeded');
+                    alert('Max files exceeded');
+                    this.removeFile(file);
+                });
+
+                this.on('sending', (file, xhr, formData) => {
+                    console.log('dropzone.sending');
+                    formData.append('_token', $('meta[name=\'csrf_token\']').attr('content'));
+                    formData.append('storage', storage);
+                    formData.append('group', groups);
+                });
+
+                this.on('removedfile', file => {
+                    console.log('dropzone.removedfile');
+                    if (objHas(file, 'data.id')) {
+                        $(`${dropname} .files-${file.data.id}`).remove();
+                        if (!isMediaLibrary) {
+                            axios
+                                .delete(platform.prefix(`/systems/files/${file.data.id}`), {
+                                    storage: $('#post-attachment-dropzone').data('storage'),
```

Comment:
We can not specify a previously created variable?
```js
 storage:storage
 //or
 storage 
```

Reply:
Yep, we can use it too. But I am not paying attention for the quality for now cuz this fix just fixing the main logic. I am working on rewrute the upload field fully.